pkg/plugins/golang/v4/scaffolds/internal/templates/test/e2e/test.go

```diff
@@ -147,6 +155,22 @@ const validatingWebhookChecksFragment = `It("should have CA injection for valida
 
 `
 
+const conversionWebhookChecksFragment = `It("should have CA injection for conversion webhooks", func() {
```

Comment:
That is great 🥇 
However, since we can have many APIs and in this way many e2e tests to check the injection of each case scenario, should not be nice add in the It something like `should have C! injection for the {{ .Resource.Kind }} conversion webhook `

Reply:
will add this info

Reply:
done